MathLib/mechanics_invariants.h

```diff
@@ -0,0 +1,166 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#ifndef MATHLIB_MECHANICS_INVARIANTS_H
+#define MATHLIB_MECHANICS_INVARIANTS_H
+
+#include <Eigen/Dense>
+
+namespace MathLib
+{
+namespace Mechanics
```

Comment:
Will be moved in MaterialLib/SolidModels/KelvinVector.h as discussed. Generalization iff this would avoid duplication.